modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SpliceAI_conf.pm

```diff
@@ -0,0 +1,195 @@
+        output_file_name        => 'spliceai_scores_',
+
+        pipeline_wide_analysis_capacity => 100,
+
+        pipeline_db => {
+            -host   => $self->o('hive_db_host'),
+            -port   => $self->o('hive_db_port'),
+            -user   => $self->o('hive_db_user'),
+            -pass   => $self->o('hive_db_password'),
+            -dbname => $ENV{'USER'} . '_hive_' . $self->o('pipeline_name'),
+            -driver => 'mysql',
+        },
+    };
+}
+
+sub resource_classes {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::resource_classes},
+        '4Gb_8c_job'  => {'LSF' => '-n 8 -q production-rh74 -R"select[mem>4000]  rusage[mem=4000]" -M4000' },
+        '4Gb_job'     => {'LSF' => '-q production-rh74 -R"select[mem>4000] rusage[mem=4000]" -M4000'},
+    };
+}
+
+sub pipeline_analyses {
+  my ($self) = @_;
+  my @analyses;
+  push @analyses, (
+      # pre run checks, directories exist etc
+
+      {   -logic_name => 'init_files',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+          -input_ids  => [{}],
+          -parameters => {
+            'input_directory' => $self->o('input_directory'),
+            'inputcmd'        => 'find #input_directory# -type f -name "all_snps_ensembl_38_*.vcf" -printf "%f\n"',
+          },
+          -flow_into  => {
+            '2->A' => {'split_files' => {'vcf_file' => '#_0#'}},
+            'A->1' => ['get_chr_dir'],
+          },
+      },
+      { -logic_name => 'split_files',
+        -module => 'Bio::EnsEMBL::Variation::Pipeline::SpliceAI::SplitFiles',
+        -input_ids  => [],
+        -parameters => {
+          'main_dir'          => $self->o('main_dir'),
+          'input_directory'   => $self->o('input_directory'),
+          'tmp_split_vcf_dir' => $self->o('tmp_split_vcf_dir'),
+          'split_vcf_dir'     => $self->o('tmp_split_vcf_input_dir'),
+          'output_dir'        => $self->o('tmp_output_dir'),
+          'step_size'         => $self->o('step_size'),
+          'check_transcripts' => $self->o('check_transcripts'),
+          # 'transcripts_file'  => $self->o('transcripts_file'),
+          'registry'          => $self->o('registry'),
+        },
+      },
+      { -logic_name => 'get_chr_dir',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+        -input_ids  => [],
+        -parameters => {
+          'input_dir' => $self->o('tmp_split_vcf_input_dir'),
+          'inputcmd'  => 'ls #input_dir#',
+        },
+        -flow_into => { 
+          '2->A' => {'init_spliceai' => {'input_chr_dir' => '#_0#'}},
+          'A->1' => ['init_merge_files'],
+        },
+      },
+      {   -logic_name => 'init_spliceai',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+          -input_ids  => [],
+          -parameters => {
+            'input_dir' => $self->o('main_dir') . '/tmp_split_vcf_input/' . '#input_chr_dir#',
```

Comment:
Can you use $self->o('tmp_split_vcf_input'). But keep in mind that we want to rename tmp_split_vcf_input to splitted_vcf_no_header_dir.